CodeStarterCamp_Week2/main.swift

```diff
@@ -46,5 +48,28 @@ func printResult(_ correctNumber: [Int]) {
     }
 }
 
-let pickedLottoNumber: Set<Int> = pickLottoNumber()
-printResult(compareTwoNumbers(myLottoNumber: myLottoNumber, pickedLottoNumber: pickedLottoNumber))
+func saveLottoNumberAndRound(lottoNumber: Set<Int>) {
+    lottoRound += 1
+    pickedLottoNumbers["\(lottoRound)회차"] = lottoNumber
+}
+
+func printNthRoundLottoNumber() {
```

Comment:
N회차의 로또 번호를 프린트해준다! 이런 네이밍으로 유추했어요!
- 해당 회차가 유동적으로 변한다면 매개변수를 활용해보는 것은 어떨까요?
```suggestion
func printLottoNumber(from round: String) {
```